
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


/home/users/delon/aemulusnu_massfunction/make_figs.py:76: IntegrationWarning: The maximum number of subdivisions (50) has been achieved.
  If increasing the limit yields no improvement it is advised to analyze 
  the integrand in order to determine the difficulties.  If the position of a 
  local difficulty can be determined (singularity, discontinuity) one will 
  probably gain from splitting up the interval and calling the integrator 
  on the subranges.  Perhaps a special-purpose integrator should be used.
  res, err = quad(dσ2dk, 0, np.inf)
  0%|          | 0/16 [00:00<?, ?it/s]/home/users/delon/aemulusnu_massfunction/make_figs.py:76: IntegrationWarning: The maximum number of subdivisions (50) has been achieved.
  If increasing the limit yields no improvement it is advised to analyze 
  the integrand in order to determine the difficulties.  If the position of a 
  local difficulty can be determined (singularity, discontinuity) one will 
  probably gain from splitting up the interval and calling the integrator 
  on the subranges.  Perhaps a special-purpose integrator should be used.
  res, err = quad(dσ2dk, 0, np.inf)
/home/users/delon/aemulusnu_massfunction/make_figs.py:116: IntegrationWarning: The maximum number of subdivisions (50) has been achieved.
  If increasing the limit yields no improvement it is advised to analyze 
  the integrand in order to determine the difficulties.  If the position of a 
  local difficulty can be determined (singularity, discontinuity) one will 
  probably gain from splitting up the interval and calling the integrator 
  on the subranges.  Perhaps a special-purpose integrator should be used.
  res, err = quad(dσ2dRdk, 0, np.inf)
  6%|▋         | 1/16 [00:09<02:15,  9.00s/it] 12%|█▎        | 2/16 [00:18<02:06,  9.02s/it] 19%|█▉        | 3/16 [00:27<01:57,  9.03s/it] 25%|██▌       | 4/16 [00:36<01:48,  9.04s/it] 31%|███▏      | 5/16 [00:45<01:39,  9.07s/it] 38%|███▊      | 6/16 [00:54<01:30,  9.08s/it] 44%|████▍     | 7/16 [01:03<01:22,  9.12s/it] 50%|█████     | 8/16 [01:12<01:13,  9.13s/it] 56%|█████▋    | 9/16 [01:21<01:03,  9.13s/it] 62%|██████▎   | 10/16 [01:31<00:55,  9.18s/it] 69%|██████▉   | 11/16 [01:40<00:45,  9.18s/it] 75%|███████▌  | 12/16 [01:49<00:36,  9.18s/it] 81%|████████▏ | 13/16 [01:58<00:27,  9.17s/it] 88%|████████▊ | 14/16 [02:07<00:18,  9.18s/it] 94%|█████████▍| 15/16 [02:17<00:09,  9.17s/it]100%|██████████| 16/16 [02:26<00:00,  9.17s/it]100%|██████████| 16/16 [02:26<00:00,  9.14s/it]
Traceback (most recent call last):
  File "/home/users/delon/aemulusnu_massfunction/make_figs.py", line 487, in <module>
    inv_weighted_cov = {a:np.linalg.inv(weighted_cov[a]) for a in weighted_cov}  
  File "/home/users/delon/aemulusnu_massfunction/make_figs.py", line 487, in <dictcomp>
    inv_weighted_cov = {a:np.linalg.inv(weighted_cov[a]) for a in weighted_cov}  
  File "<__array_function__ internals>", line 200, in inv
  File "/home/users/delon/.local/lib/python3.9/site-packages/numpy/linalg/linalg.py", line 532, in inv
    _assert_stacked_2d(a)
  File "/home/users/delon/.local/lib/python3.9/site-packages/numpy/linalg/linalg.py", line 183, in _assert_stacked_2d
    raise LinAlgError('%d-dimensional array given. Array must be '
numpy.linalg.LinAlgError: 0-dimensional array given. Array must be at least two-dimensional
